ironfish-rust/src/frost_utils/split_secret.rs

```diff
@@ -77,39 +70,18 @@ mod test {
     use crate::{keys::SaplingKey, test_util::create_multisig_identities};
     use ironfish_frost::frost::{frost::keys::reconstruct, JubjubBlake2b512};
 
-    #[test]
-    fn test_invalid_secret() {
-        let identities = create_multisig_identities(10);
-
-        let vec = vec![1; 31];
-        let config = SecretShareConfig {
-            min_signers: 2,
-            identities,
-            secret: vec,
-        };
-
-        let rng = rand::thread_rng();
-        let result = split_secret(&config, rng);
-        assert!(result.is_err());
-        assert!(
-            matches!(result.unwrap_err().kind, IronfishErrorKind::InvalidSecret),
-            "expected InvalidSecret error"
-        );
-    }
```

Comment:
Why was this removed?

Reply:
because this was removed: 
```
let secret_bytes: [u8; 32] = config
        .secret
        .clone()
        .try_into()
        .map_err(|_| IronfishError::new(IronfishErrorKind::InvalidSecret))?;
```

we can keep the map_err if that is preferred

Reply:
we're no longer passing in a vector. the earlier conversion to the vector was unnecessary. This edge case handled at the napi layer i believe

Reply:
https://github.com/iron-fish/ironfish/blob/rahul/ifl-2307-create-new-gsk-when-splitting-secret-from-dealer/ironfish-rust-nodejs/src/frost.rs#L204